src/decision/src/GpsDecision.cpp

```diff
@@ -1,9 +1,9 @@
 /*
- * Created By: Gareth Ellis and Chris Chen
- * Created On: September 22, 2016
+ * Created By: Gareth Ellis
+ * Created On: May 18, 2017
  * Description: The Decision Node for GPS, takes in a point relative to
  *              the robots location and heading and broadcasts a
- *              recommended twist message
+ *              recommended twist message. Revised to use TF's
  */
```

Comment:
Same thing with this file as with `GPSManager.cpp`, probably want to just review it as if I were submitting it for the first time